pandas/core/frame.py

```diff
@@ -2199,8 +2199,10 @@ def to_feather(self, path, **kwargs) -> None:
 
         Parameters
         ----------
-        path : str
-            String file path.
+        path : str or file-like object
+            If a string, it will be used as Root Directory path. By file-like
+            object, we refer to objects with a write() method, such as
+            a file handler (e.g. via builtin open function) or io.BytesIO.
```

Comment:
do we describe what file-like means elsewhere?  id be ok with leaving that out.

backticks around `io.BytesIO`

Reply:
we do describe it in other writer methods and in the user guide - I took it out  (fb37d73)